mcs/class/corlib/corlib.dll.sources

```diff
@@ -1636,12 +1636,17 @@ corert/ThreadPoolBoundHandle.cs
 
 ../../../external/corert/src/Runtime.Base/src/System/Runtime/InteropServices/NativeCallableAttribute.cs
 
+../../../external/corert/src/System.Private.CoreLib/shared/System/Tuple.cs
 ../../../external/corert/src/System.Private.CoreLib/shared/System/TupleExtensions.cs
 ../../../external/corert/src/System.Private.CoreLib/shared/System/ValueTuple.cs
 
 ../../../external/corert/src/System.Private.CoreLib/shared/System/Collections/DictionaryEntry.cs
+#../../../external/corert/src/System.Private.CoreLib/shared/System/Collections/ObjectEqualityComparer.cs
```

Comment:
looks like this file was removed, why keep it commented out?

Reply:
Forgot to remove it